spec/fixtures/resolvable_with_strict_mode/a.rb

```diff
@@ -0,0 +1,3 @@
+require_relative 'c'
+
+class A < C; end
```

Comment:
These fixture files should probably all end with a trailing EOL

Reply:
Yep, agree - done.